### PR TITLE
Add catalog concept + working demo data / credentials

### PR DESCRIPTION
**Pull request by jrmgx**
We decided to add the concept of catalog in the documentation 
as well as giving a working set of data (catalog + credentials)

This PR reflect that.

Please take some time to read it, it is mainly to spot typos
or make remark about readability 

Thank you

### CODE REVIEW

File: _pages/iOS/objc.md [hunk 1]
```diff
@@ -129,42 +136,74 @@ You need to grant camera permission before initializing environment
...
+| 93 | Bed Bath | Table |
+| 92 | Target | Outdoor Chair |
+| 91 | Target | Chair |
+| 90 | Restoration Hardware | Table |
+| 89 | Restoration Hardware | Chair |
+| 86 | FINLANDEK | Couch |
+| 84 | Whirlpool | Fridge |
+| 83 | Samsung | Washer |
+| 82 | Lenovo | Yoga 900 |
+| 81 | Samsung | TV |
```
Comment (sgara) [inline _pages/iOS/objc.md]: It's a detail but that should go to a _data file or something, in order to have it in one place and ease maintenance

File: _pages/iOS/objc.md [hunk 2]
```diff
@@ -129,42 +136,74 @@ You need to grant camera permission before initializing environment
...
 
 Use `AGTProductsDataController` in order to query products over **Augment** database.
 If the product exists in **Augment** database, `AGTProductsDataController` returns an `AGTProduct` instance within completion block
 You also can check if a query for given identifier had done before via `AGTProductsDataController` as seen below:
 
 ```objc
-- (void)loadDummyModel {
-    AGTProduct *cachedProduct = [self.augmentSDK.productsDataController productForIdentifier:DummyProductIdentifier];
+- (void)loadMyModel {
+    AGTProduct *cachedProduct = [self.augmentSDK.productsDataController productForIdentifier: @"84"]; // From the catalog
```
Comment (sgara) [inline _pages/iOS/objc.md]: Same thing here

File: _pages/iOS/objc.md [hunk 3]
```diff
@@ -64,12 +66,18 @@ You can now use the main 3 following classes:
...
-
 ## Setup
 
 You need `AGTAugmentSDK` and `AGTView`. You may also need `AGTDevice` if you want to control device features like flash or autofocus
 
 ```objc
+// These are demo credentials linked to the demo catalog below
+NSString* APP_ID = @"357fee36746668573ceb2f5957c4869ee1a62a112639bac9b0fae43c7c431692";
+NSString* APP_SECRET = @"80ae1420e164e0440d5329067bcdd953e9fa6c63b75c001c06d169a4f11268c5";
+NSString* VUFORIA_KEY = @"Dl5pzxoca2JxLcYxBJ2pIeIE4dNcK0etMeb1746L7lq6vSFen43cS7P1P/HXjwHtUouV5Xus2U0F7WHUTKuO629jKFO13fBQczuY52UJcSEhsu9jHPMaupo5CpqQT3TFTQjlhzHhVXiVMEqq7RI+Edwh8TCSfGAbNRdbIELTfK+8YDYqwEHDbp62mFrs68YnCEQZDrpcLyC8WzFCVZtnUq3Cj3YBUfQ6gNnENYiuLf06gAAF/FcaF65VYveGRBbp3hpkqolX28bxPiUYNVknCSFXICPHciVntxF+rcHW5rrX7Cg/IVFGdNRF";
```
Comment (sgara) [inline _pages/iOS/objc.md]: There too : please put the values in a _data file

File: _pages/iOS/objc.md (hunk 1 above)
Comment (jrmgx) [inline _pages/iOS/objc.md]: I'll put everything in a _data.yml file 
thank for the suggestion

**Comment by jrmgx** [comment]
As there is no text change on the javascript documentation I removed @mmorisset from the reviewers
I will merge this PR now that @sgara suggestion are done

**Pull request merged**